test/functional/rpc_psbt.py

```diff
@@ -78,17 +78,17 @@ def test_psbt_incomplete_after_invalid_modification(self):
         node = self.nodes[2]
         wallet = node.get_wallet_rpc(self.default_wallet_name)
         address = wallet.getnewaddress()
-        wallet.sendtoaddress(address=address, amount=1.0)
+        wallet.sendtoaddress(address=address, amount=Decimal("1.0"))
```

Comment:
is this really better than before?
```suggestion
        wallet.sendtoaddress(address=address, amount=1)
```
or
```suggestion
        wallet.sendtoaddress(address=address, amount="1")
```

Both seem to work